src/Components/SvaFinder.js

```diff
@@ -2,8 +2,9 @@ import { render, Component } from 'preact';
 import t from 'Utility/i18n';
 import { fetchSvaDataBySlug } from '../Utility/companies';
 import PropTypes from 'prop-types';
+import deepmerge from 'deepmerge';
```

Comment:
I don't know how large that lib is but we've previously only used that for dev iirc. :/

Reply:
Oh, you're right.

> UMD bundle is 723B minified+gzipped